web/app/embed.js

```diff
@@ -56,6 +56,56 @@ function init() {
 
   setTimeout(postHashToIframe, 1000);
 
+  const userInfo = {
+    node: null,
+    back: null,
+    init(user) {
+      if (!this.node) {
+        this.node = document.createElement('div');
+        this.node.style = `position: fixed; top: 0; right: 0; bottom: 0;width: 400px; transform: translate(400px, 0); transition: transform 0.4s ease-out;`;
```

Comment:
400px could brake for mobile users. Even newest iPhone X has only 375 effective points in width.
https://www.paintcodeapp.com/news/ultimate-guide-to-iphone-resolutions